vimPlugins.volt: fix pname

M3L6H opened the pull request:
The nvzone plugins were named with an `nvzone` prefix due to how generic the individual plugin names are ([ref](https://github.com/NixOS/nixpkgs/pull/357120)). This caused their pnames to also have an `nvzone` prefix. However, plugin managers like Lazy.nvim expect pname to match the name of the git repository, hence the override of pname in this PR.

## Things done

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

PerchunPak reviewed (changes requested):
`nvzone-` prefix was added because most of the plugins had very generic names (see #357120 for discussion). If there is a problem with `pname`, please override it in `overrides.nix`

M3L6H committed:
(vimPlugins.nvzone-volt): fix pname

The nvzone plugins were named with an `nvzone` prefix due to how generic
the individual plugins are: https://github.com/NixOS/nixpkgs/pull/357120
This caused their pnames to also have an `nvzone` prefix. However,
plugin managers like Lazy.nvim expect pname to match the name of the git
repository, hence the override of pname in this commit.

khaneliman commented:
> However, plugin managers like Lazy.nvim expect pname to match the name of the git repository, hence the override of pname in this PR.

Should we proactively apply that to the other plugins with prefixes that don't match their upstream repo?

PerchunPak reviewed (commented):
Could you do this for all nvzone plugins please?

M3L6H commented:
> Could you do this for all nvzone plugins please?

Sure

M3L6H committed:
(vimPlugins.nvzone-typr): fix name

The nvzone plugins were named with an `nvzone` prefix due to how generic the individual plugins are: NixOS#357120 This caused their pnames to also have an `nvzone` prefix. However, plugin managers like Lazy.nvim expect pname to match the name of the git repository, hence the override of pname in this commit.

M3L6H committed:
(vimPlugins.nvzone-menu): fix pname

The nvzone plugins were named with an `nvzone` prefix due to how generic the individual plugins are: NixOS#357120 This caused their pnames to also have an `nvzone` prefix. However, plugin managers like Lazy.nvim expect pname to match the name of the git repository, hence the override of pname in this commit.

M3L6H committed:
(vimPlugins.nvzone-minty): fix pname

The nvzone plugins were named with an `nvzone` prefix due to how generic the individual plugins are: NixOS#357120 This caused their pnames to also have an `nvzone` prefix. However, plugin managers like Lazy.nvim expect pname to match the name of the git repository, hence the override of pname in this commit.
